Cargo.toml

```diff
@@ -41,3 +41,4 @@ rustls = ["reqwest/rustls-tls"]
 
 [dev-dependencies]
 httpmock = "0.4"
+serial_test = "0.5"
```

Comment:
@LionsAd  This only compiles when building for testing, and it allows us to put multiple Gaggle tests in a single integration test file.